test-client/Cargo.toml

```diff
@@ -19,16 +19,16 @@ hash-db = "0.15.2"
 hex = "0.4"
 serde = "1.0.55"
 serde_json = "1.0.55"
-sc-client-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1" }
-sc-client-db = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1", features = ["test-helpers"]}
-sc-consensus = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1" }
-sc-executor = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1" }
-sc-light = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1" }
-sc-service = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1", default-features = false, features = ["test-helpers"]}
-sp-blockchain = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1" }
-sp-consensus = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1" }
-sp-core = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1" }
-sp-keystore = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1" }
-sp-keyring = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1" }
-sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1" }
-sp-state-machine = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.24-1" }
+sc-client-api = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.29" }
+sc-client-db = { version = "0.10.0-dev", git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.29", features = ["test-helpers"]}
```

Comment:
same version thing, why no need to specify version before and you add now.

Reply:
fixed